ansible/roles/test/tasks/arpall.yml

```diff
@@ -261,11 +261,9 @@
   always:
     # Recover DUT interface IP Address before entering this test case
     - name: restore dut original state
-      command: config reload -y
-      become: yes
-
-    - name: wait 60 seconds for ports to be up
-      pause: seconds=60
+      include: "roles/test/tasks/common_tasks/reload_sonic.yml"
```

Comment:
For clarity, maybe this file should be renamed "reload_config.yml" or "reload_sonic_config.yml"

Reply:
@jleveque done: reload_sonic.yml -> reload_config.yml